include/private/gcconfig.h

```diff
@@ -1083,7 +1083,6 @@ extern int etext[], _end[];
 #  define DATASTART PTR_ALIGN_UP((ptr_t)etext, 0x1000)
 #  define DATAEND ((ptr_t)_end)
 #  define DYNAMIC_LOADING
-#  define MPROTECT_VDB
```

Comment:
If MPROTECT_VDB is no longer defined, then there is no need of Serenity-specific changes in code inside ifdef MPROTECT_VDB: i.e. please remove defined(SERENITY) in os_dep.c line 3263 and 3381.

Reply:
Thanks, updated!